laravel-app/app/Models/PostLike.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * @property int $user_id
+ * @property int $post_id
+ */
+class PostLike extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user_id',
+        'post_id',
+    ];
+
+    public $timestamps = false;
```

Comment:
Why did you set $timestamps to false?

Reply:
It was set to false so that the inserts will not automatically create timestamps at the table, I did not include this since we will not be using the timestamps for likes, as of now.

Reply:
Is there created_at & updated_at columns in your table for post_likes table?

Reply:
None sir

Reply:
okay, i guess you can exclude this declaration in your next PR since you dont have timestamps on the table.

LGTM 🚀